.chloggen/print-config-redaction.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```
I doubt it will break many people, but this is still realistically a breaking change (even behind a feature gate) since we're renaming the command.